generate-figures-tables.sh

```diff
@@ -25,8 +19,21 @@ script_dir=${BASEDIR}/scripts
 
 # Ensure output directories exist
 mkdir -p ${BASEDIR}/figures/pngs
+mkdir -p ${BASEDIR}/figures/pdfs
 mkdir -p ${BASEDIR}/tables
 
+# check if s3 files directory already exists
+# if it does, delete it
+if [ -d "${BASEDIR}/s3_files" ]; then
+  rm -r ${BASEDIR}/s3_files
+fi
+mkdir -p ${BASEDIR}/s3_files
+
+# copy over any files needed from S3
```

Comment:
since we're deleting the directory, it's all of them!
Also, since these steps take a long time, maybe let's print a status message...

```suggestion
# copy over files from S3
echo "Downloading data files..."
```